dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcCompletenessTableManager.java

```diff
@@ -256,7 +256,12 @@ private String getPartitionClause(AnalyticsTablePartition partition) {
 
   private List<AnalyticsTableColumn> getColumns() {
     String idColAlias = "concat(ds.uid,'-',ps.iso,'-',ous.organisationunituid,'-',ao.uid) as id ";
-    String timelyDateDiff = "cast(cdr.date as date) - ps.enddate";
+    int secondsInDay = (int) (24L * 60 * 60);
```

Comment:
We could extract it into a constant or move it to DateUtils.java (see DAYS_IN_YEAR and MS_PER_DAY, for example)